kitsune/sumo/static/sumo/scss/form-wizard.scss

```diff
@@ -0,0 +1,45 @@
+:root,
+:host {
+  --color-progress: var(--color-blue-06);
+}
+
+:host {
+  display: flex;
+  flex-direction: column;
+  border-radius: 8px;
+  box-shadow: 0px 2px 6px rgba(58, 57, 68, 0.2);
+}
+
+.form-wizard-content {
+  padding-inline: 32px;
+  padding-block-start: 24px;
+}
+
+h2 {
+  margin: 0;
+  font-size: 1.5rem;
+  line-height: 1;
+  color: var(--color-heading);
+}
+
+ul {
+  margin-inline-end: 48px;
+}
+
+section {
+  display: flex;
```

Comment:
It looks like this needs to be rebased on top of your BaseFormStep stuff - but we probably want to have those BaseFormStep's flex too, if they're not already doing that. Maybe with a class or something?

Reply:
For easier tracking (since this all got moved to a different file) - I've changed the rule to:

```css
section,
::slotted([slot="active"])  {
    display: flex;
    flex: 1;
}
```

Let me know if you had something else in mind!